### DISCUSSION TITLE
Notification Providers

### DISCUSSION THREAD

**Discussion by reply-mj** [fluxcd/flux2#2934] [Q&A]
I am installing Flux into an AWS EKS Cluster (dozens of clusters actually) and I want to be able to send notification to an AWS Service such as SNS or CloudWatch.

Is there and example somewhere that I can see please?

**Comment by kingdonb** [comment]
It is possible via the `generic` webhook provider, however we do not have good end-to-end documentation for how to do this. The hard part IMHO is on the AWS side, you will need to create a lambda or something that exposes itself as a webhook, that Flux can call in order to trigger the notification. (The notification details are in the webhook body, and your lambda would have to do some work to translate them into the target SNS topic or CloudWatch event.)
